bin/classes/com/hp/ppm/integration/rally/RallyIntegrationConnector.properties

```diff
@@ -0,0 +1,73 @@
+BASE_URL=Base URL
```

Comment:
We should not include ANY file from /bin folder. 

You should add .bin in .gitignore.